File: app/src/main/java/com/example/bottlerocketstudioscodingtest/UI/MainActivity.java
package com.example.bottlerocketstudioscodingtest.UI;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.bottlerocketstudioscodingtest.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            StoresFragment storeList = new StoresFragment();

            getSupportFragmentManager().beginTransaction()
                    .add(R.id.store_list_fragment, storeList)
                    .commit();
        }
    }
}

File: app/src/main/java/com/example/bottlerocketstudioscodingtestkotlin/UI/MainActivity.kt
package com.example.bottlerocketstudioscodingtestkotlin.UI

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.example.bottlerocketstudioscodingtestkotlin.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            val storeList = StoresFragment()

            supportFragmentManager.beginTransaction()
                    .add(R.id.store_list_fragment, storeList)
                    .commit()
        }
    }
}